main.go

```diff
@@ -40,6 +34,19 @@ func main() {
 			Name:   "debug",
 			EnvVar: "DEBUG",
 		},
+		cli.BoolFlag{
+			Name:   "api",
```

Comment:
Shouldn't these be something like `enable-api` to be more descriptive, as it's a flag not a "command"?

Reply:
sure, good idea